service/paxos/paxos_state.cc

```diff
@@ -366,11 +368,104 @@ static sstring paxos_state_cf_filter(const schema& s, const schema& state_schema
         : sstring();
 }
 
-paxos_store::paxos_store(db::system_keyspace& sys_ks)
-: _sys_ks(sys_ks)
+static const sstring paxos_state_table_suffix = "_scylla_paxos_state";
```

Comment:
Let's use "$paxos" to make it shorter and to use the character traditionally designated for internal stuff.

Reply:
Do you mean as a prefix `$paxos_<base_table_name>` or suffix is ok?

Reply:
I means base$paxos, it's nice to get the names grouped when sorted.

Reply:
OK, will update.